src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/VideoFanTakeover.java

```diff
@@ -111,51 +100,60 @@ public Double getCurrentVideoTimeOnMobile(String slotName) {
     return Double.parseDouble(result);
   }
 
-  public double getAdSlotHeight(String slotName) {
-    return driver.findElement(By.cssSelector(AdsContent.getSlotSelector(slotName))).getSize().getHeight();
+  public double getAdSlotHeight(String slotSelector) {
+    return driver.findElement(By.cssSelector(slotSelector)).getSize().getHeight();
   }
 
   public double getAdVideoHeight(String slotName) {
     return driver.findElement(By.cssSelector(String.format(UI_ELEMENT_SELECTOR_FORMAT, slotName))).getSize().getHeight();
   }
 
   public void verifyFandomTabOpened(String tabUrl) {
-    Assertion.assertEquals(tabUrl, FANDOM_URL);
+    Assertion.assertEquals(tabUrl, AD_REDIRECT_URL);
   }
 
   public boolean isVideoAdBiggerThanImageAdOasis(double videoHeight, double imageHeight) {
     int percentResult = (int)Math.round(100-(100/(videoHeight/imageHeight)));
     if (percentResult == PERCENTAGE_DIFFERENCE_BETWEEN_VIDEO_AND_IAMGE_AD) {
       return true;
     }
+    PageObjectLogging.log("isVideoAdBiggerThanImageAdOasis",
+            "Expected percentage difference between video height and image height is not equal with "
+            + PERCENTAGE_DIFFERENCE_BETWEEN_VIDEO_AND_IAMGE_AD + " percent", false, driver);
     return false;
   }
 
 // Different way of checking slot sizes on mercury because of the very small difference between two slots sizes
-  public boolean isVideoAdBiggerThanImageAdMercury(double videoHeight, double imageHeight) {
+  public boolean isVideoAdBiggerThanImageAdMobile(double videoHeight, double imageHeight) {
     if (videoHeight > imageHeight) {
       return true;
     }
+    PageObjectLogging.log("isVideoAdBiggerThanImageAdMobile",
+            "Ad image height = " + imageHeight + " is not smaller than video height = " + videoHeight, false, driver);
     return false;
   }
 
-  public boolean isImageAdInCorrectSize(double imageHeight, String slotName) throws InterruptedException {
+  public boolean isImageAdInCorrectSize(double imageHeight, String slotSelector) throws InterruptedException {
     long time = System.currentTimeMillis();
-    long endTime = time+2000;
+    long endTime = time+5000;
```

Comment:
but then we're increasing time of bad-scenario test pass by 3secs.
I'm wondering if sign that animation didn't finished in 2secs shouldn't be a sign that something's wrong, what do you think?